plutus-tx-plugin/executables/profile/Main.hs

```diff
@@ -8,12 +9,30 @@
 {-# OPTIONS_GHC -fplugin-opt PlutusTx.Plugin:debug-context #-}
 {-# OPTIONS_GHC -fplugin-opt PlutusTx.Plugin:profile-all #-}
 
--- | Executable for profiling. Add the program you want to profile here.
--- Plugin options only work in this file so you have to define the programs here.
+{- | Executable for profiling. See note [Profiling instructions]-}
+
+{- Note [Profiling instructions]
+Work flow for profiling evaluation time:
+1. Compile your program with profile-all and dump-plc
+2. Run the dumped program with uplc --trace-mode LogsWithTimestamps --log-output logs
+3. Run logToStacks filePaths and input it to flamegraph.pl
+
+Running @logToStacks@ gives you the program's .stacks file.
+@logToStacks@ takes file paths (of the dumped programs) as input.
+
+To get a flamegraph, you need to have flamegraph.pl from
+https://github.com/brendangregg/FlameGraph/.
+Input your program's .stacks file to flamegraph.pl to get a flamegraph. E.g.
+$ ~/FlameGraph/flamegraph.pl < plutus-tx-plugin/executables/profile/fib4.stacks > fib4.svg
+
+4. Run firefox yourFile.svg
+flamegraph.pl turns the stacks into a .svg file. You can view the file with a browser. E.g.
+$ firefox fib4.svg
+ -}
 
 module Main where
```

Comment:
I think this thing no longer really belongs here: it could live in `plutus-core`, since it's something that you use to process the output of `uplc`! But probably leave it for now.